internal/corazawaf/body_buffer.go

```diff
@@ -25,6 +25,7 @@ type BodyBuffer struct {
 
 // Write appends data to the body buffer by chunks
 // You may dump io.Readers using io.Copy(br, reader)
+// TODO(jcchavezs): Stop writing beyond the limit
```

Comment:
We won't need this TODO if we remove `RequestBodyWriter` which I think we should.